source/integrations/index.html

```diff
@@ -103,60 +147,163 @@
-      if (location.hash !== '' && location.hash.indexOf('#search/') === 0) {
-        // set default value in search from URL
-        jQuery('.component-search input').val(decodeURIComponent(location.hash).substring(8));
+      //add these to the window for dynamic grabbing in matchSelectToURL()
+      window.$catFilter = $('.integration_filter select[data-id="cat"]');
+      window.$versionFilter = $('.integration_filter select[data-id="version"]');
+      window.$iot_classFilter = $('.integration_filter select[data-id="iot_class"]');
+      window.$quality_scaleFilter = $('.integration_filter select[data-id="quality_scale"]');
+      window.$searchFilter = $('.component-search input');
+
+      //listen for changes in the hash / user clicking back or forward button in browser
+      window.addEventListener('hashchange', applyFilter);
+      window.addEventListener('popstate', applyFilter);
+
+      //on initial page load, if there's no active filters set the default to featured
+      let query = buildQueryFromURL();
+      if (Object.keys(query).length === 0) {
+        updateQueryURL('cat', 'featured');
       }
+      applyFilter();
 
       // add focus to the search field - even on IE
       setTimeout(function () {
-        jQuery('.component-search input').focus();
+        $('.component-search input').focus();
       }, 1);
     }
     init();
 
-    /**
-     * filter all components, based on the location's hash and render them into the component box
-     */
-    function applyFilter() {
+    // get query value pair that matches provided id from url. // for buildQueryFromURL()
+    function getQueryValue(id) {
+      let index = id.length + 2; // id + '#' and '/' => '#search/'
+      if (id === 'cat') index = 1;
+
+      // get value from #hash url
+      let query = decodeURIComponent(location.hash).substring(index).toLowerCase();
+
+      // get value from ?query url if it's present
+      let queryParams = new URLSearchParams(location.search);
+      if (queryParams.has(id)) {
+        query = queryParams.get(id);
+      }
+
+      return query;
+    }
+
+    // build out query object of active filters for applyFilter()
+    function buildQueryFromURL() {
+      let queryFilter = {};
+      let hash = location.hash || '';
+      let queryParams = new URLSearchParams(location.search);
+
+      if (hash.indexOf('#all') === 0 || queryParams.get('cat') === 'all') {
+        queryFilter.cat = 'all';
+      }
+
+      if (hash.indexOf('#featured') === 0 || queryParams.get('cat') === 'featured') {
+        // only show those with featured = true
+        queryFilter.cat = 'featured';
+      }
+
+      if (hash.indexOf('#search/') === 0 || queryParams.has('search')) {
+        let query = getQueryValue('search');
+        queryFilter.search = query;
+      } else {
+        // reset search box when not searching
+        $searchFilter.val(null);
+      }
+      
+      if (hash.indexOf('#version/') === 0 || queryParams.has('version')) {
```

Comment:
```suggestion
      if (hash.startsWith('#version/') || queryParams.has('version')) {
```
The same goes for the rest of these

Reply:
updated all instances of `indexOf` to use `startsWith` method